### PR TITLE
Add versionbot workflow

### PR DESCRIPTION
**Pull request by klutchell**

### CODE REVIEW

**Comment by ab77** [comment]
@klutchell to clear/modify branch protection rules (e.g.):

```
org=balenaci
repo=vb-test
branch=master

curl -X PUT https://api.github.com/repos/$org/$repo/branches/$branch/protection \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer $GITHUB_ACCESS_TOKEN" \
  https://api.github.com/repos/$org/$repo/branches/$branch/protection \
  -d '{"required_status_checks":{"strict":true,"contexts":[]},"enforce_admins":false,"required_pull_request_reviews":{"dismissal_restrictions":{"users":[],"teams":[]},"dismiss_stale_reviews":false,"require_code_owner_reviews":false,"required_approving_review_count":1,"bypass_pull_request_allowances":{"users":[],"teams":[]}},"restrictions":{"users":[],"teams":[],"apps":[]},"required_linear_history":false,"allow_force_pushes":true,"allow_deletions":false,"block_creations":false,"required_conversation_resolution":false}'
```

See, https://docs.github.com/en/rest/branches/branch-protection#update-branch-protection

**Comment by ab77** [comment]
@balena-ci retest

**Comment by klutchell** [comment]
@balena-ci I self-certify!

**Comment by klutchell** [comment]
@ab77 we need *something* to be required for bulldozer to run. Can we mark just `VersionBot/generate-version` as required for now? I wish we had admin access like I requested.

**Comment by klutchell** [comment]
@balena-ci rebase

**Comment by klutchell** [comment]
@balena-ci retest

File: .github/actions/docker/action.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - name: Run SUT container
+      working-directory: ${{ inputs.working_directory }}
+      shell: bash
+      run: |
+        if [[ ! -z "${COMPOSE_VARS}" ]]; then
+          echo $COMPOSE_VARS | base64 --decode > .env
+        fi
+        docker-compose \
+          -f docker-compose.yml \
+          -f docker-compose.test.yml \
```
[the remark this answers is not in the document]
Comment (ab77) [reply .github/actions/docker/action.yml]: @klutchell probably just need to ack this review...

**Review by ab77 — APPROVED** [review]

**Pull request merged**